components/esp_hw_support/port/esp32c3/regi2c_ctrl.h

```diff
@@ -42,7 +42,7 @@ uint8_t rom_i2c_readReg_Mask(uint8_t block, uint8_t host_id, uint8_t reg_add, ui
 void rom_i2c_writeReg(uint8_t block, uint8_t host_id, uint8_t reg_add, uint8_t data);
 void rom_i2c_writeReg_Mask(uint8_t block, uint8_t host_id, uint8_t reg_add, uint8_t msb, uint8_t lsb, uint8_t data);
 
-#ifdef BOOTLOADER_BUILD
+#if defined(BOOTLOADER_BUILD) && !defined(CONFIG_MCUBOOT)
```

Comment:
Is this necessary? Does it cause build warnings?

Reply:
Exactly!